lib/solargraph/complex_type/unique_type.rb

```diff
@@ -68,6 +68,7 @@ def items
       # @return [String]
       def to_rbs
         "#{namespace}#{parameters? ? "[#{subtypes.map { |s| s.to_rbs }.join(', ')}]" : ''}"
+      # "
```

Comment:
Forgive me this weirdness - Emacs is not smart enough to parse the expression above it and thinks a runaway double-quoted string starts here.  The commented-out double quote seems enough to let it handle the rest of the file!

Reply:
Heheh. VS Code has similar problems with certain string syntaxes. I have a feeling it's an unavoidable limitation of trying to identify Ruby string ranges with a regular expression.